src/line-chart/line-series.component.ts

```diff
@@ -42,6 +42,15 @@ import { sortLinear, sortByTime, sortByDomain } from '../utils/sort';
         [class.active]="isActive(data)"
         [class.inactive]="isInactive(data)"
       />
+     <svg:g ngx-charts-line
```

Comment:
Also, the area component has `opacity` and `fill` inputs already, so we can use those.